ops/model.py

```diff
@@ -335,8 +335,9 @@ def get_secret(self, *, id: str | None = None, label: str | None = None) -> Secr
     def get_cloud_spec(self) -> CloudSpec:
         """Get details of the cloud in which the model is deployed.
 
-        Note: This information is only available for machine charms,
-        not Kubernetes sidecar charms.
+        .. jujuchanged:: 3.6.10
+            This information is available on both machine charms and Kubernetes
+            charms. With earlier Juju versions, it was only available on machine charms.
 
         Returns:
             a specification for the cloud in which the model is deployed,
```

Comment:
Noticed while looking through `ModelError` raising docs: there's a "Raises" note a few lines below this that should also be removed or updated ("if called in a Kubernetes model" should probably be "if called without trust").